build-langs

```diff
@@ -79,6 +79,7 @@ for %langs{@langs}:p -> (:key($name), :value(%lang)) {
 
             when 'ALGOL 68'    { $ver = "Algol 68 Genie $digits" }
             when 'AWK'
+               | 'Bash'
```

Comment:
Unlike say AWK which is a language with many implementations, there's only one Bash, it doesn't make sense to prefix it with GNU as there's no confusion about which version we're running. Please remove this line.